### PR TITLE
AP-6769 Regression: 24/7 calendar timetable name lost

### PR DESCRIPTION
**Pull request by shapon2004**

### CODE REVIEW

File: Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/app/provider/entries/resource/fields/ResourceTimetable.js [hunk 1]
```diff
@@ -22,8 +22,7 @@ module.exports = function(bpmnFactory, elementRegistry, translate, options) {
 
       timetablesWithNoEmptyName.forEach(function(timetable) {
         timetableOptions.push({
-          name: timetable.default && timetable.name !== translate('arrivalTimetable.name') ?
-            timetable.name + ' ' + translate('arrivalTimetable.name') : timetable.name,
+          name: timetable.name == 'Log timetable' ? translate('logtimetable.name') : timetable.name,
```
Comment (lahiru-maha) [inline Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/app/provider/entries/resource/fields/ResourceTimetable.js]: Should 'Log Timetable' be in a constant?

File: Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/app/provider/entries/simulationInfo/fields/TimeTableField.js [hunk 2]
```diff
@@ -22,8 +22,8 @@ module.exports = function (bpmnFactory, elementRegistry, translate) {
 
       timetablesWithNoEmptyName.forEach(function (timetable) {
         timetableOptions.push({
-          name: timetable.name,
-          value: timetable.id
+           name: timetable.name == 'Log timetable' ? translate('logtimetable.name') : timetable.name,
```
Comment (lahiru-maha) [inline Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/app/provider/entries/simulationInfo/fields/TimeTableField.js]: Should 'Log Timetable' be in a constant?

File: Apromore-Custom-Plugins/Process-Simulation-Info-Logic/src/main/java/org/apromore/processsimulation/service/SimulationInfoService.java [hunk 3]
```diff
@@ -204,14 +206,18 @@ private void deriveTimetable(
 
         CalendarModel calendarModel = simulationData.getCalendarModel();
 
-        if (calendarModel.getName().equals(SimulationData.DEFAULT_CALENDAR_NAME)) {
```
Comment (lahiru-maha) [inline Apromore-Custom-Plugins/Process-Simulation-Info-Logic/src/main/java/org/apromore/processsimulation/service/SimulationInfoService.java:207]: Likely can get rid of the Constant `SimulationData.DEFAULT_CALENDAR_NAME`

Comment (shapon2004) [reply Apromore-Custom-Plugins/Process-Simulation-Info-Logic/src/main/java/org/apromore/processsimulation/service/SimulationInfoService.java:207]: Keeping this one SimulationData.DEFAULT_CALENDAR_NAME. Remove the one i have created.

**Review by lahiru-maha — APPROVED** [review]
LGTM

**Pull request merged**